src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9175,6 +9175,13 @@ public sealed partial class RequiresUnreferencedCodeAttribute : System.Attribute
         public string Message { get { throw null; } }
         public string? Url { get { throw null; } set { } }
     }
+	[System.AttributeUsageAttribute(System.AttributeTargets.Parameter | System.AttributeTargets.Field | System.AttributeTargets.Property)]
```

Comment:
some tabs crept in here. could you please instead do ref file updates with the tool?
https://github.com/dotnet/runtime/blob/2879e6ca886eb084335ccd0a579019fbe359628e/docs/coding-guidelines/updating-ref-source.md

Reply:
The tool does not work for me :/
 
`dotnet build --no-incremental /t:GenerateReferenceSource`
![grafik](https://user-images.githubusercontent.com/10373452/146993237-bf766cd7-1010-43cc-9ad1-e2ced877dd6c.png)

Reply:
I indented it manually

Reply:
hmm, the error is cut off so not sure what that is, a binary mismatch most likely. I wonder whether it would work if you used the dotnet.cmd in the src folder.